synaptic_reconstruction/tools/util.py

```diff
@@ -168,3 +174,67 @@ def _available_devices():
         else:
             available_devices.append(device)
     return available_devices
+
+
+def get_current_tiling(tiling: dict, default_tiling: dict, image_shape):
+    # get tiling values from qt objects
+    for k, v in tiling.items():
+        for k2, v2 in v.items():
+            if isinstance(v2, int):
+                continue
+            tiling[k][k2] = v2.value()
+    # check if user inputs tiling/halo or not
+    if default_tiling == tiling:
+        if len(image_shape) == 2:
```

Comment:
We need to rethink this and rather set the 2d tiling size depending on the model then the image shape.
It's easiest to discuss in person.